### PR TITLE
Adding file ADOPTERS.md

### LINKED ISSUE #215: Add an ADOPTERS.md page
Project e2e-framework is used and loved by many in the Kubernetes community. We should provide a place where folks can let others know that they are using the project.

### PR DESCRIPTION
**Pull request by vladimirvivien**
Adding file ADOPTERS.md and its first adopter.

Fixes #215

### CODE REVIEW

**Comment by vladimirvivien** [comment]
@harshanarayana PTAL

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# E2E-Framework Adopters
+
+Project e2e-framework is used and loved by many in the Kubernetes community. 
+While we are not aware of everyon using the project, below is a list of adopters (and/or contributors) who have
```
Comment (harshanarayana) [inline ADOPTERS.md:4]:
```suggestion
While we are not aware of everyone using the project, below is a list of adopters (and/or contributors) who have
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# E2E-Framework Adopters
+
+Project e2e-framework is used and loved by many in the Kubernetes community.
```
Comment (harshanarayana) [inline ADOPTERS.md:3]:
```suggestion
This project is used and loved by many in the Kubernetes community. 
```

Comment (harshanarayana) [reply ADOPTERS.md:3]: Please check if this makes it much easier to read ?

File: ADOPTERS.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# E2E-Framework Adopters
+
+Project e2e-framework is used and loved by many in the Kubernetes community. 
+While we are not aware of everyon using the project, below is a list of adopters (and/or contributors) who have
+agreed to share how they use the framework in their porjects.
+
+### Adding your story
+
+If you are using the e2e-framework to test your Kubernetes components, and you want to share your story, feel free to open a PR and update this document.
```
Comment (harshanarayana) [inline ADOPTERS.md:9]:
```suggestion
If you are using the `e2e-framework` to test your Kubernetes components, and you want to share your story, feel free to open a PR and update this document.
```

**Review by harshanarayana — COMMENTED** [review]
@vladimirvivien  Thanks for adding this one. Been thinking about this for a while now. 

Left a few minor tweaks.

**Comment by negz** [comment]
LGTM on behalf of @crossplane. 🙂 

Someone shared this query with me, which I found helpful to get a sense of how widely used the framework is:

https://github.com/search?type=code&q=lang%3Ago++NOT+repo%3Akubernetes-sigs%2Fe2e-framework+NOT+is%3Afork+%22sigs.k8s.io%2Fe2e-framework%22&p=2

**Comment by harshanarayana** [comment]
@negz  That is a pretty nifty query. Thanks for the link.

File: ADOPTERS.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## E2E-Framework adopters
+
+### Crossplane
+[Crossplane](https://github.com/crossplane/crossplane) has ported all of their integration tests to use e2e-framework for their Crossplane project.
+See their work [here](https://github.com/crossplane/crossplane/pull/4187)
+
```
Comment (maruina) [inline ADOPTERS.md:16]:
```suggestion
[Datadog](https://github.com/DataDog/) has ported their e2e tests from the [Kubernetes e2e test framework](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/framework) to the e2e-framework. The e2e-framework is also used to test different controllers.
```

**Review by vladimirvivien — COMMENTED** [review]
@harshanarayana @maruina @negz I updated the info you provided. PTAL.

**Pull request merged**